cohortextractor/tpp_backend.py

```diff
@@ -2805,6 +2805,35 @@ def patients_with_value_from_file(
     def patients_which_exist_in_file(self, f_path):
         return self.patients_with_value_from_file(f_path)
 
+    def patients_with_an_isaric_record(
+        self,
+        returning,
+        between=None,
+        date_filter_column=None,
```

Comment:
This is unusual but makes sense to me, assuming there are multiple date columns in the ISARIC data that we'd want to filter on. Is there an obvious "primary" date column we could use as a default here? Or is it better to force people to be explicit?

Reply:
I'm following @alexwalkerepi suggested api here: https://github.com/opensafely-core/cohort-extractor/issues/614

Reply:
:+1: Makes sense